### DISCUSSION TITLE
Eslint: Fix error  auto imported Vue api 'not defined'

### DISCUSSION THREAD

**Discussion by ciarantr** [nuxt/nuxt#16324] [Questions]
Hi, is it possible to fix eslint linting errors for auto imported apis ref, computed etc. ?

**Comment by JackEdwardLyons** [comment]
Any news on this? How do we configure eslint or Nuxt to handle auto imports?

<img width="392" alt="Screen Shot 2022-11-22 at 9 23 10 am" src="https://user-images.githubusercontent.com/16175039/203177851-d0849e0d-9c0d-4e0f-92bc-2dc9e9c59a32.png">
